src/main/scala/mesosphere/marathon/tasks/TaskTracker.scala

```diff
@@ -5,6 +5,7 @@ import javax.inject.Inject
 import mesosphere.marathon.MarathonConf
```

Comment:
We put a lot of effort into TaskTracker recently. Does it make sense to extract a trait and move it into core/*?